docs/5g-on-helium/5g-poc.mdx

```diff
@@ -0,0 +1,191 @@
+- **Stage 2: ** 5G Mappers Proof of Coverage
+- **Stage 3: ** Additional Methods of Proof of Coverage
+
+---
+
+## Current 5G PoC Reward Elegibility
+
+MOBILE Rewards are earned by a Wallet with a 5G Hotspot and at least one connected, SAS registered,
+CBRS Small Cell Radio in one "lump sum" every roughly 24-hour interval. This system allows a high
+tolerance for downtime in infrastructure that is beyond the 5G Hotspot Owner's control.
+
+To qualify for the Genesis Rewards in a Reward Period, a CBRS Small Cell Radio must:
+
+- Be connected to a 5G Hotspot.
+- Be properly installed.
+  - Be registered with the SAS (Outdoor Radios Only).
+- Broadcast at least 1 "heartbeat" per Reward Period.
+
+:::note What is a "Heartbeat"?
+
+A "heartbeat" is an authorization that allows the Small Cell Radio to transmit 5G coverage.
+
+Heartbeats occur every 60 seconds and transmit authorizations are valid for 240 seconds, this
+rolling overlap ensures that a Small Cell Radio can broadcast on an ongoing basis while allowing for
+brief interruptions.
+
+:::
+
+---
+
+## Stage 1: Genesis Rewards
+
+:::info Reward Targets
+
+During Genesis, 100M Mobile tokens are allocated to be issued per day. Given the different Reward
+Transactions triggers, daily rewards may not equate to an exact 100M MOBILE.
+
+Check out this
+[Example Reward Transaction](https://explorer.helium.com/txns/0n4TO_2W14rrsGt-nxBTotiHFCFukkO04QoXIxaOtAY)
+for reference.
+
+:::
+
+### Genesis Phase 1: Time Period Triggered Rewards
+
+> - Reward Period is 1440 blocks and closely aligns with 00:00 UTC to 23:59 UTC.
+> - Reward submission occurs approx 17 hours later, once a day, around 17:00 UTC.
+
+At this Milestone, to qualify for the Genesis Rewards in a Reward Period, a CBRS Small Cell Radio
+must:
+
+- Be connected to a 5G Hotspot.
+- Be properly installed.
+  - Be registered with the SAS (Outdoor Radios Only).
+- Broadcast at least 1 "heartbeat" per Reward Period.
+
+### Genesis Phase 2: Consensus Election Triggered Rewards
+
+> - Reward Period is 48 Consensus Elections (1440 blocks, approximately 24 hours).
+> - Reward submission occurs approx 30 minutes after the Reward Period ends.
+
+At this Milestone, to qualify for the Genesis Rewards in a Reward Period, a CBRS Small Cell Radio
+must:
+
+- Be connected to a 5G Hotspot.
+- Be properly installed.
+  - Be registered with the SAS (Outdoor Radios Only).
+- Broadcast at least 1 "heartbeat" per Reward Period.
+
+### Genesis Phase 3: Speed Test and Quality of Service
+
+> - 5G Hotspots perform periodic speed tests which will be considered in reward calculations.
+> - A minimum Download speed, Upload speed, and Latency will be required by each 5G Hotspot to
+>   ensure it is providing quality coverage.
+
+A 5G Hotspot will perform a speed test every 12 hours with the results being averaged over the
+previous 6 tests.
+
+To meet the speed test requirement a 5G Hotspot must have a rolling average Download speed of at
```

Comment:
Need a link to HIP53 and mention that 100/10 requirement is specified in it.

Reply:
link added